rook/ensure.go

```diff
@@ -32,12 +34,21 @@ func (pcx *PackageContext) EnsureDependencies(ctx context.Context, forceUpdate b
 	pcx.Package.Vendor = filepath.Join(pcx.Package.LocalPath, "dependencies")
 
 	for _, dependency := range pcx.AllDependencies {
-		errInner := pcx.EnsurePackage(dependency, forceUpdate)
-		if errInner != nil {
-			print.Warn(errors.Wrapf(errInner, "failed to ensure package %s", dependency))
+		r := retrier.New(retrier.ConstantBackoff(2, 100*time.Millisecond), nil)
+		err := r.Run(func() error {
+			print.Verb("attempting to ensure dependency", dependency)
+			errInner := pcx.EnsurePackage(dependency, forceUpdate)
+			if errInner != nil {
+				print.Warn(errors.Wrapf(errInner, "failed to ensure package %s", dependency))
+				return errInner
+			}
+			print.Info(pcx.Package, "successfully ensured dependency files for", dependency)
+			return nil
+		})
+		if err != nil {
+			print.Warn("failed to ensure package '", dependency, "' after 2 attempts, skipping")
```

Comment:
If this is intended to ignore the error ("skipping") then log the error and set `err` to `nil`. Otherwise, let it be returned.